app/controllers/landmarks_controller.rb

```diff
@@ -10,6 +10,7 @@ def index
   def show
     @landmark = Landmark.find(params[:id])
     @review = Review.new
+    @landmark.average_review = @landmark.average_rating
```

Comment:
@Salisbum Never mind--I see now that you have the method on the model that does the calculation and here you're setting the value on the table. You could consider not persisting the information if you're going to recalculate it on every `show` pageload anyway, but I'll let you decide that.